python/lsst/analysis/tools/actions/plot/gridPlot.py

```diff
@@ -84,10 +84,7 @@ class GridPlot(PlotAction):
         doc="Independent data definitions. The key of this dict is the panel ID. The values are keys of data "
         "to plot (comma-separated for multiple) where each key may be a subset of a full key.",
     )
-    figsize = ListField[float](
-        doc="Figure size.",
-        default=[8, 8],
-    )
+    figsize = ListField[float](doc="Figure size.", default=[8, 8], length=2)
```

Comment:
Can this be expanded out to match the other fields formatting?

Reply:
@czwa I'm not sure I understand, do you just mean the code formatting? Or to change the other fields to further constrain their formatting?

Reply:
Just code formatting, and I prefer the expanded "one element per line" version.